subcommands/common_config.go

```diff
@@ -71,6 +72,8 @@ func SetConfig(opts *SetConfigOptions) {
 }
 
 type LogConfigsOptions struct {
+	Api           *client.Api
+	Factory       string
```

Comment:
I guess this is okay. I suspect volodya will have stronger opinions though. In my mind I was thinkink you could just have a:
```
UserLookup map[string]User
```
This can then be nil so that volodya's request to make it optional stays possible in our API.

Then you could add a simple helper function to this file like like `GetUsersLookup(api, factory) map[string]User` and then the users of this struct have an easy way to populate it.